src/core/MOM_open_boundary.F90

```diff
@@ -1194,6 +1194,27 @@ subroutine initialize_obc_tides(OBC, US, param_file)
       "Fixed reference date to use for nodal modulation of boundary tides.", &
       default=0, do_not_log=tides)
 
+  ! The following parameters are to be obsolete in the future
```

Comment:
These 4 parameters would need to have default arguments if they are to be properly logged in the correct MOM_parameter_doc files.

It may be that we need to add support for a default array of integers to `get_param_integer_array()`, analogously to what we recently added for `get_param_real_array()`.

If we were to get community buy-in on the idea of moving to use the non-OBC parameters for these tidal parameters, we might deal with this by storing the values set above, adding `do_not_log=.true.` to these `get_param()` calls, and then directly adding a warning if variables like `OBC_TIDE_ADD_EQ_PHASE` are set differently from `TIDE_USE_EQ_PHASE`.  I will try to raise this at this evening's MOM6 dev call, and then provide further guidance depending on how that conversation goes.

Reply:
It would be more complicated if we want to add a warning if the old variables (e.g., `OBC_TIDE_ADD_EQ_PHASE`) are set differently from the new ones (e.g., `TIDE_USE_EQ_PHASE`), but I don't think this should happen if both astronomical and OBC tidal forcings are turned on. I did not add a `do_not_log` option because these parameters were used by OBC tidal forcing only, and I think it would be easier to remind the users of OBC tidal forcing that these parameters will be obsolete.